raster/src/main/scala/geotrellis/raster/geotiff/GeoTiffReprojectRasterSource.scala

```diff
@@ -155,6 +155,8 @@ class GeoTiffReprojectRasterSource(
 
   def convert(targetCellType: TargetCellType): RasterSource =
     GeoTiffReprojectRasterSource(dataPath, crs, resampleTarget, resampleMethod, strategy, targetCellType = Some(targetCellType))
+
+  override def toString: String = s"GeoTiffReprojectRasterSource(${dataPath.value},$crs,$resampleTarget,$resampleMethod)"
```

Comment:
GeoTiffReprojectRasterSource will have an additional parameter `OverviewStrategy` in #3200. Should we add that to this method as well?